### PR TITLE
No more graphs with only zeroes

### PR DESCRIPTION
**Pull request by PijusOz**
When there aren't any data, statistics won't show up.

### CODE REVIEW

File: front-end-side/src/Pages/Data.js [hunk 1]
```diff
@@ -71,102 +71,112 @@ const EnterData = () => {
     return (
         <div>
             <Navbar />
+            {
+                newExpenses !== 0 ?
+                    (
+                        <>
```
Comment (dissco6) [inline front-end-side/src/Pages/Data.js:77]: What's that?

File: front-end-side/src/Pages/Data.js [hunk 2]
```diff
@@ -71,102 +71,112 @@ const EnterData = () => {
     return (
         <div>
             <Navbar />
+            {
+                newExpenses !== 0 ?
```
Comment (dissco6) [inline front-end-side/src/Pages/Data.js:75]: We are getting _Upload your files or enter data to get statistics..._ for a second even if we have uploaded data. Could we fix it?

File: front-end-side/src/Pages/Data.js [hunk 3]
```diff
@@ -71,102 +71,112 @@ const EnterData = () => {
...
+                                ) : (
+                                    null
+                                )}
                             </div>
-                        </div>
+                        </>
                     ) : (
-                        null
+                        <>
+                            <div className="container" style={{ marginTop: "30px" }}>
```
Comment (dissco6) [inline front-end-side/src/Pages/Data.js:166]: This block (modals etc.) is repeated, isn't it possible to make this update without repeating code?

File: front-end-side/src/Pages/Data.js (hunk 1 above)
Comment (PijusOz) [reply front-end-side/src/Pages/Data.js:77]: Did it

File: front-end-side/src/Pages/Data.js [hunk 4]
```diff
@@ -71,102 +71,112 @@ const EnterData = () => {
...
+                                    </div>
+                                ) : (
+                                    null
+                                )}
                             </div>
-                        </div>
+                        </>
                     ) : (
-                        null
+                        <>
```
[the remark this answers is not in the document]
Comment (PijusOz) [reply front-end-side/src/Pages/Data.js:165]: ez

File: front-end-side/src/Pages/Data.js (hunk 3 above)
Comment (PijusOz) [reply front-end-side/src/Pages/Data.js]: Fixed it

**Pull request merged**